Library/Homebrew/rubocops/formula_desc_cop.rb

```diff
@@ -0,0 +1,68 @@
+module RuboCop
+  module Cop
+    module Homebrew
+      class FormulaDesc < Cop
+        def on_class(node)
+          return unless processed_source.path =~ %r{#{ENV["HOMEBREW_LIBRARY"]}\/Taps\/homebrew}
+          class_node, parent_class_node, body = *node
+          formula_name = class_node.const_name
+          return unless parent_class_node && parent_class_node.const_name == "Formula" && body
+          check(node, body, formula_name)
+        end
```

Comment:
Can you move this to a helper method? We'll probably need this for every Formula cop.